packages/extension-base/ctp-extension-gcf-adapter.js

```diff
@@ -0,0 +1,24 @@
+// Google Cloud Functions
+
+const createReject = ctx => error => {
+  ctx.status(400).json({
+    errors: Array.isArray(error) ? error : [error],
+  });
+};
+
+const createAccept = ctx => (update = []) => {
+  ctx.status(200).json({
+    actions: Array.isArray(update) ? update : [update],
+  });
```

Comment:
I like GCF's API so much more.

Reply:
Agreed. From what I get, this is the `express.js` API.

Reply:
Yeah its Hapi/Express like. Maybe even more Hapi than Express. The Azure is more barebones Node Http.